### PR TITLE
gui edit fix

### PR DESCRIPTION
**Pull request by libbyhopfauf**
fixes edit mode for the gui.

### CODE REVIEW

**Comment by libbyhopfauf** [comment]
Sorry guys, I didn't mean to automatically merge the branch!

File: vrecord [hunk 1]
```diff
@@ -798,18 +798,20 @@ shift $(( ${OPTIND} - 1 ))
 
 if [[ "${runtype}" = "GUI" ]] ; then
     _master_gui
-elif [[ "${runtype}" = "reset" ]] ; then
+fi
+if [[ "${runtype}" = "reset" ]] ; then
     _report -q -n "Reseting the configuration will clear ${config_file}. Please enter [Y] to confirm: "
     read reset_response
-    if [[ "${reset_response}" = [Yy] ]] ; then
+    if [[ "${reset_response}" = "[Y]" ]] ; then
```
Comment (privatezero) [inline vrecord]: I think this should still be "[Yy]"

File: vrecord [hunk 2]
```diff
@@ -798,18 +798,20 @@ shift $(( ${OPTIND} - 1 ))
...
         _report -d "Clearing ${config_file}."
         echo -n "" > "${config_file}"
         runtype="edit"
     else
         _report -d "Reset aborted. Exiting."
         exit 0
     fi
-elif [[ "${runtype}" = "edit" ]] ; then
+fi
+if [[ "${runtype}" = "edit" ]] ; then
```
Comment (privatezero) [inline vrecord]: I think this all could have been kept as a `if elif fi` rather than changing it back into two `if fi` `if fi` blocks

Comment (libbyhopfauf) [reply vrecord]: when i tested it that way, it said that there was a disagreement

File: vrecord (hunk 1 above)
Comment (libbyhopfauf) [reply vrecord]: Okay, I wasn't sure since on an older version that I was comparing it to it was the way that i changed it to.

Comment (retokromer) [reply vrecord]: Yes, see https://github.com/amiaopensource/vrecord/pull/202#pullrequestreview-83068360.

**Comment by retokromer** [comment]
Thank you very much, @libbyhopfauf!

I‘ll wait until this evening (CET) and then merge, if no opposition.

**Comment by dericed** [comment]
tested locally and it fixes the issue. Thx @libbyhopfauf.

**Comment by retokromer** [comment]
Thank you, @libbyhopfauf and @dericed !

**Comment by privatezero** [comment]
Yay! Thank you @libbyhopfauf!

**Pull request merged**